tests/fail/tokio_mvp.rs

```diff
@@ -0,0 +1,7 @@
+//@compile-flags: -Zmiri-disable-isolation
+//@error-pattern: can't call foreign function: epoll_create1
+//@normalize-stderr-test: "   = note: inside .*\n" -> ""
+//@only-target-linux: the errors differ too much between platforms
+
+#[tokio::main]
+async fn main() {}
```

Comment:
Btw would be nice to also have a 'pass' test for https://github.com/rust-lang/miri/issues/602#issuecomment-884019764.

Should we have these tests of non-stdlib crates in a subdir in our test folders? The root dirs are already quite messy, so something like `{pass,fail}/crates` or so might make sense?

Reply:
yea grouping them into a custom folder makes sense.

Reply:
https://github.com/rust-lang/miri/pull/2408 does these changes.